app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+
+
+def copy_file(command: str) -> None:
+    parts = command.split()
```

Comment:
Code Efficiency: It's a good practice to check if the command starts with 'cp' before splitting it. This can save some processing time when the command is not 'cp'.